sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -517,7 +517,6 @@ def merge_accumulators(self, accumulators, *args, **kwargs):
               (holds_comparables, result_heap),
               comparable.value if holds_comparables else comparable)
 
-    assert result_heap is not None and holds_comparables is not None
```

Comment:
Not sure I follow why we need the changes beyond `.pre-commit-config.yaml` - is there a reason we need to change `combiners.py` and `core.py`

Reply:
Oops, I think that got carried over from something else. Sorry.